pkgs/servers/rmfakecloud/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, fetchFromGitHub, buildGoModule }:
+
+buildGoModule rec {
+  pname = "rmfakecloud";
+  version = "unstable-20220118";
+
+  src = fetchFromGitHub {
+    owner = "ddvk";
+    repo = pname;
+    rev = "ac9c9c4838496205f5372705b36b484233754ad5";
+    sha256 = "sha256-lSRLuqa4LbafG4Rh0yP9FT5NIjmX9SqNdLpddy46nIk=";
+  };
+
+  vendorSha256 = "sha256-kLwa5ucCzZ8MYIyiWUYIYMw7yqJ/RkmGtHgxjz1MH48=";
+
+  postPatch = ''
+    # skip including the JS SPA, which is difficult to build
```

Comment:
For what it's worth I was able to build it with the JS thingie:
http://cgit.notk.org/asmadeus/nixos-config.git/tree/pkgs/rmfakecloud/default.nix

I don't use it though so don't particularly care and will switch to this package regardless on this gets in, but it shouldn't be much different.

Reply:
Thanks! I think this should be made in a separate subsequent PR to this one.

Reply:
Definitely agree there, this is already great -- no reason to hold this PR back :)
I'll be happy to submit the UI if it turns out it's useful, I actually haven't even looked at it yet...